nrf_wifi/fw_if/umac_if/inc/fw/host_rpu_sys_if.h

```diff
@@ -796,7 +796,8 @@ enum op_band {
 	BAND_24G
 };
 
-#define TWT_EXTEND_SP_EDCA  0x1
+#define TWT_EXTEND_SP_EDCA  (1 << 1)
```

Comment:
I meant prefixing for all `RPU_FEAT`, it should be an enum, you can fix existing code in a new commit and then add this feature.

```
#ifndef BIT
#define BIT(x) (1 << x)
#endif

enum rpu_feature_flags {
   RPU_FEAT_TWT_EXTEND_SP_EDCA    = BIT(1),
   RPU_FEAT_DISABLE_DFS_CHANNELS = BIT(2),
};
```

Reply:
Updated